superset-frontend/src/components/Menu/Menu.jsx

```diff
@@ -59,6 +59,28 @@ const StyledHeader = styled.header`
     flex-direction: column;
     justify-content: center;
   }
+
+  .navbar-nav > li > a {
+    &:after {
+      content: '';
+      position: absolute;
+      bottom: -3px;
+      left: 0;
+      width: 100%;
+      height: 3px;
+      background-color: #20a7c9;
```

Comment:
```suggestion
      background-color: ${({ theme }) => theme.colors.primary.base};
```
And actually, just so everyone reading this is aware, there's an extra benefit of doing it this way, which I haven't really demonstrated to folks.... 
Since the `supersetTheme` is currently applied at the top of the `App` using a `ThemeProvider`, we get to (eventually) expand this with some JS to merge a user's preferences/config onto the theme... something like so:
`<ThemeProvider theme={customizeTheme(supersetTheme, userThemePrefs)}><App /></ThemeProvider>` and/or employ any number of pre-rolled themes, e.g. `<ThemeProvider theme={supersetTheme(dark)}><App /></ThemeProvider>`